### ISSUE TITLE
Flip panel issue

### ISSUE THREAD

**Issue by G-Guy** [BEEmod/BEE2.4#1440]
Hello. I have an issue with one of default items, the flip panel. When I am in test editor, the panels are set to black side by me. However when I build the test they are white side for some reason. Disabling or enabling start reversed in flip panel options doesn't change anything. I would like to note that this issue appeared ONLY after I started using BEE2.4. It wasn't present without it installed.

Item icon - https://freeimage.host/i/KWlE6g
Editor screenshot - https://freeimage.host/i/KWljup
In-game screenshot - https://freeimage.host/i/KWlwwN

Please do something about it.

**Comment by seagemgames** [comment]
For item issues, use the BEEmod/BEE2-items repo located here: https://github.com/BEEmod/BEE2-items/issues
And also, this is a duplicate of # 3467 in BEE2-items https://github.com/BEEmod/BEE2-items/issues/3467

**Comment by LautaroL20** [comment]
Like seagemgames said, you have posted in the wrong section. Anything related to items & styles must be posted on the item´s page. Also, this issue will be fixed in the next release.
